packages/terra-worklist-data-grid/src/WorklistDataGrid.module.scss

```diff
@@ -0,0 +1,47 @@
+// Themes
+@import './orion-fusion-theme/WorklistDataGrid.module';
+@import './clinical-lowlight-theme/WorklistDataGrid.module';
+
+:local {
+  .container {
+    height: 100%;
+  }
+
+  [role='grid'] [tabIndex='0']:focus {
+    background-color: var(--terra-worklist-data-grid-cell-focus-background-color, #cbe7fa);
+    box-shadow: var(--terra-worklist-data-grid-cell-focus-box-shadow, inset 0 0 0 1px rgba(38, 162, 229, 0.5), inset 0 0 0 4px rgba(76, 178, 233, 0.2));
+    outline: none;
+  }
+
+  .worklist-data-grid {
+    // Style for the grid itself
+    border: var(--terra-worklist-data-grid-border, 1px solid #dedfe0);
+    border-collapse: collapse;
+    margin: 1em auto;
+    width: 100%;
+  }
+
+  .worklist-data-grid-column-header {
+    background-color: var(--terra-worklist-data-grid-column-header-background-color, #dedfe0);
+    border: var(--terra-worklist-data-grid-column-header-border, 1px solid #000);
+    color: var(--terra-worklist-data-grid-column-header-text-color);
+  }
+
+  .worklist-data-grid-row-header {
+    background-color: var(--terra-worklist-data-grid-row-header-background-color, #dedfe0);
+    border: var(--terra-worklist-data-grid-row-header-cell-border, 1px solid  #000);
+    color: var(--terra-worklist-data-grid-row-header-text-color);
+  }
+
+  .worklist-data-grid-cell-data {
+    // Style for the table cell data
+    border: var(--terra-worklist-data-grid-cell-border, 1px solid #dedfe0);
+    height: 100%;
+    position: relative;
+  }
+
+  [role='grid'] .striped {
```

Comment:
Wouldn't we want this to be a descendant of the worklist class instead of using the grid role?

Reply:
As we changed the implementation to even-odd selectors, this section the code became moot and is removed.